chacra/tests/test_util.py

```diff
@@ -446,3 +446,32 @@ def test_field_override(self, session):
         result = util.get_distributions_file_context("ceph")
         assert "foo" in result["data"]
         assert result['data']['foo'] == "blarg"
+
+
+class TestRepositoryIsDisabled(object):
+
+    def test_nothing_is_configured(self):
+        assert util.repository_is_disabled('foo') is False
+
+    def test_project_explicitly_disabled(self):
+        pecan.conf.repos = {'foo': {'disabled': True}}
+        assert util.repository_is_disabled('foo') is True
+
+    def test_unconfigured_repo_with_disabled_repos(self):
+        pecan.conf.disable_unconfigured_repos = True
+        pecan.conf.repos = {'bar': {'disabled': True}}
+        assert util.repository_is_disabled('foo') is False
```

Comment:
I'm confused by this test. Wouldn't ``foo`` actually be disabled here because it's not configured?